### DISCUSSION TITLE
about loss and optimizer

### DISCUSSION THREAD

**Discussion by wangkwok** [mrdbourke/pytorch-deep-learning#599] [General]
Dear @mishazawa,
Firstly great thanks for your tutorial, it is really very good especially for me, a beginner of ML. Thank you very much!
I have one question about Pytorch. I do not understanding that the loss and optimizer seem NOT link to the model, however only the result y_logits, and label y_train can get the gradient, backward propagation and even update the model parameters? it is so funny!
![2023-08-11_110758](https://github.com/mrdbourke/pytorch-deep-learning/assets/16315196/4fa97874-d235-4122-980a-10e9ea39e80a)
